app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+class CommandError(Exception):
+    def __str__(self) -> str:
+        return "Your command is not complete"
+
+
+def copy_file(command: str) -> None:
+    try:
+        copy_command, old_file, new_file = command.split(" ")
+    except CommandError:
+        raise CommandError
```

Comment:
The `except` block is incorrectly catching `CommandError`. Since `command.split(" ")` can raise a `ValueError` if the command does not split into exactly three parts, you should catch `ValueError` here instead. Modify the `except` clause to handle `ValueError` and then raise `CommandError`.